docs/pages/access-controls/reference.mdx

```diff
@@ -33,10 +34,26 @@ To manage cluster roles, a Teleport administrator can use the Web UI or the
 command line using [tctl resource commands](../setup/reference/resources.mdx).
 To see the list of roles in a Teleport cluster, an administrator can execute:
 
+<ScopedBlock scope={["oss", "enterprise"]}>
+
 ```code
+# Log in to your cluster so you can use tctl.
```

Comment:
```suggestion
# Log in to your cluster with tsh so you can use tctl.
```

Let's be extra explicit that the login is with tsh so that people don't mistake this for an error.